vcd/resource_vcd_org.go

```diff
@@ -53,24 +54,25 @@ func resourceOrg() *schema.Resource {
 				Description: "True if this organization is enabled (allows login and all other operations).",
 			},
 			"deployed_vm_quota": &schema.Schema{
-				Type:        schema.TypeInt,
-				Optional:    true,
-				Default:     -1,
-				Description: "Maximum number of virtual machines that can be deployed simultaneously by a member of this organization.",
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      0,
```

Comment:
Doesn't we released that? and this change impacts back compatibility?

Reply:
No. Using `-1` was wrong. It's a bug.

`-1` is an illegal value, which was silently converted to 0. It will impact compatibility, but in the way that a bug does. We are fixing what was wrong in the first place.
Also the GUI does not accept `-1`, and the API gives an error on such a value.